src/components/PeopleManagement/_PeopleManagement.scss

```
@@ -21,4 +21,10 @@
   border-radius: 50%;
   padding: 10px;
   color: white;
+}
+.learner-detail-section {
+  box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.175);
+  padding-top: 2rem; 
+  width: 25rem;
```

I know the 1px seems negligible but because it combines with the margin and it adds more if you try it on local 
```suggestion
  padding-bottom: 1px;  
  width: 25rem;
```